segmentation_models_pytorch/decoders/pan/decoder.py

```diff
@@ -220,7 +247,7 @@ def __init__(
                 upscale_mode=upscale_mode,
             )
 
-    def forward(self, *features):
+    def forward(self, features: List[torch.Tensor]) -> torch.Tensor:
```

Comment:
This is a pretty big change. You went from `model(x1, x2, x3)` to `model([x1, x2, x3])`. Is this intentional, or was this an accidental type hint change?

Reply:
This was an intentional change for `torchscript` compatibility. Yeah, while the model interface does not change, decoder interface has been changed, so might break smth for those who use building blocks

Reply:
I'm fine with that if it's only the building blocks that changed and not the outward-facing encoders/decoders.

Reply:
I hope it will be fine as well. While Decoders themselves are not private, they are not advertised as a public API. The main use case is the `model` API, which is fully backward compatible.